plugins/modules/discovery_intent.py

```diff
@@ -812,6 +896,9 @@ def handle_global_credentials(self, response=None):
             if not isinstance(net_conf_port_list, list):
                 msg = "Global net Conf Ports be passed as a list"
                 self.discovery_specific_cred_failure(msg=msg)
+            if response.get("netconfCredential") is None:
+                msg = "No Global Net Conf Ports are present in the Cisco Catalyst Center"
```

Comment:
Global netconf ports are not present ...

Reply:
Addressed